jenkins-pipelines/manager/older-enterprise-ami-manager-upgrade.jenkinsfile

```diff
@@ -7,7 +7,7 @@ managerPipeline(
     backend: 'aws',
     region: 'us-east-1',
 
-    scylla_version: '2021.1',
+    scylla_version: '2022.1',
```

Comment:
latest enterprise is `2022.2`

Reply:
Right, but these jobs are ment to run with the second to latest enterprise, hence 2022.1